src/mips/psyqo/cdrom-device.hh

```diff
@@ -321,6 +321,15 @@ class CDRomDevice final : public CDRom {
         S getState() const { return static_cast<S>(m_device->m_state); }
     };
 
+    /**
+     * @brief Checks if the CDROM device is in idle state.
+     *
+     * @details This assumes that all CDRomDeviceStateEnum have a first enum member
+     * with value 0 which corresponds to the "idle" state
+     *
+     */
+    bool isIdle() const;
```

Comment:
We might just want to have the implementation here, since it's a simple direct access. Saves a jump and results in the same amount of generated code overall.